test/javascript/packs/utils/calendar.test.js

```diff
@@ -1,4 +1,15 @@
-import { compareHours } from "../../../../app/javascript/packs/utils/calendar";
+import moment from 'moment';
+import { compareHours, prev, next, current, innerRange, startDate } from "../../../../app/javascript/packs/utils/calendar";
+
+const formattedDate = (date) => date.format('YYYY-MM-DD');
+const CURRENT_DATE = '2022-08-15'
+
+beforeAll(() => {
+  jest.useFakeTimers('modern').setSystemTime(new Date(CURRENT_DATE))
+})
+afterAll(() => {
+  jest.useRealTimers()
+})
```

Comment:
Usually, we place these hooks inside the `describe`, so I think we should leave these inside the `describe` but that's not a blocker, so I am going to approve this PR